src/index.css

```diff
@@ -9,21 +9,256 @@ ul {
 }
 
 body {
-  background-color: #050518;
+  background-color: #fafafa;
   font-family: sans-serif;
   overflow: auto;
-  color: #fff;
+  color: #000;
```

Comment:
- [ ] Well done implementing the changes required by the last CR, however, your design should match the Zeplin design. Please 🙏🏽 fix the width to avoid the horizontal scroll and fix the right margin.

| Zeplin design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/232245492-367fbb01-6689-4db0-a984-3a68089d747f.png) | ![image](https://user-images.githubusercontent.com/58642949/232245546-c5988d89-7c82-4c13-ae7a-efaf4bbc801a.png) |